pkg/config/setup/system_probe.go

```diff
@@ -173,6 +173,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 
 	// User Tracer
 	cfg.BindEnvAndSetDefault(join(diNS, "enabled"), false, "DD_DYNAMIC_INSTRUMENTATION_ENABLED")
+	cfg.BindEnvAndSetDefault(join(diNS, "resolve_inlined_program_counters"), false, "DD_DYNAMIC_INSTRUMENTATION_RESOLVE_INLINED_PROGRAM_COUNTERS")
```

Comment:
FYI if the config key exactly matches the env var name you want (with a `DD_`) prefix, then you do not need to override it. `join(diNS, "resolve_inlined_program_counters")` would resolve to `DD_DYNAMIC_INSTRUMENTATION_RESOLVE_INLINED_PROGRAM_COUNTERS` by default.

Reply:
Got it - I suppose I like that this makes where the config is set in code searchable by env variable. Can I just leave it incase the logic you describe changes at some point?

Reply:
Changing that logic would be a pretty big breaking change. It doesn't hurt anything to have it explicit like you have it.